include/git2/merge.h

```diff
@@ -162,6 +162,8 @@ typedef enum {
 	GIT_MERGE_FILE_DIFF_MINIMAL = (1 << 7),
 } git_merge_file_flag_t;
 
+#define GIT_MERGE_CONFLICT_MARKER_SIZE	7
```

Comment:
Shouldn't you also update `GIT_MERGE_FILE_OPTIONS_INIT`?

Reply:
🤔

To clarify: the current `INIT` macro initializes it to `0`, which implies this default will be used.  Functionally, this is correct.

We could leave this as-is or we could update the macro to explicitly assign this default.  I feel like in other places (eg, checkout opts) this is the pattern we use, to leave things blank / null / zero and then assign a default after the fact.

I mention this only to be explicit about this.  If you think updating this is better (more visibility by the users?) then I can change it.

Reply:
Okay, I guess that's fine. The only implication of this is that the user cannot ever set the conflict marker size to `0`. I don't know whether there's any strange usecase that would want that, though.